scripts/user_interface.py

```diff
@@ -719,15 +734,30 @@ def onEnter(*args):
                 self.tkkb.wm_attributes("-topmost", 1)
                 self.tkkb.transient(self.root)
                 self.tkkb.protocol("WM_DELETE_WINDOW", self.kill_tkkb)
-            
+
+
+    def send_print(self):
+        try:
+            conn = cups.Connection()
+            printers = conn.getPrinters()
+            default_printer = printers.keys()[1]#defaults to the first printer installed
```

Comment:
I think this should be configurable in setup.py
Ideas to do so in `setup.py`:
- call conn.getPrinters() in setup and list available printers (best)
- or, at least, create a configuration key 'printer_name' filled by the user during setup and iterate on printers.keys to find it

What do you think ?